Superlista/Models/BridgeModels/UserModelConverter.swift

```diff
@@ -102,15 +110,19 @@ class UserModelConverter {
         var sharedWithMe: [CKListModel] = []
         
         id = CKRecord.ID(recordName: user.id)
-        name = user.name!
+        name = user.name ?? ""
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()